site/compiler/test.js

```diff
@@ -20,19 +20,23 @@ describe('site/compiler', () => {
         createStateNavigator(),
       );
 
-      expect(routes.length).to.equal(12);
-      expect(routes[0].filePath).to.equal('index.html');
-      expect(routes[1].filePath).to.equal('what-we-do/index.html');
-      expect(routes[2].filePath).to.equal('our-work/index.html');
-      expect(routes[3].filePath).to.equal('about-us/index.html');
-      expect(routes[4].filePath).to.equal('about-us/join-us/index.html');
-      expect(routes[5].filePath).to.equal('about-us/events/index.html');
-      expect(routes[6].filePath).to.equal('our-work/case-study/retailer/index.html');
-      expect(routes[7].filePath).to.equal('technology/index.html');
-      expect(routes[8].filePath).to.equal('404.html');
-      expect(routes[9].filePath).to.equal('50x/index.html');
-      expect(routes[10].filePath).to.equal('offline/index.html');
-      expect(routes[11].filePath).to.equal('about-us/people/index.html');
+      const filePaths = routes.map(r => r.filePath);
+
+      expect(filePaths).to.include.members([
+        'index.html',
+        'what-we-do/index.html',
+        'our-work/index.html',
+        'about-us/index.html',
+        'about-us/join-us/index.html',
+        'about-us/events/index.html',
+        'our-work/case-study/retailer/index.html',
+        'our-work/case-study/candem/index.html',
```

Comment:
This needs to be changed to 'Camden'. I think it needs to be a bulk update, as this spelling's used elsewhere as well